test/upload-file-test.js

```diff
@@ -187,6 +188,57 @@ suite.addBatch({
                                     assert.ifError(err);
                                 }
                             },
+                            "and we get the file from web interface without login": {
+                                topic: function(doc, feed, pair, cl) {
+                                    var cred = makeCred(cl, pair),
+                                        browser = new Browser(),
+                                        callback = this.callback,
+                                        url = doc.fullImage.url;
+
+                                    browser.visit(url, function(err) {
+                                        callback(err, browser);
+                                    });
+                                },
+                                "it fails correctly": function(err, br) {
+                                    br.assert.status(403);
+                                }
+                            },
+                            "and we login for try get file": {
```

Comment:
`"and we login and try to get the file"`

Reply:
Done.